org-drill-table.el

```diff
@@ -191,11 +197,13 @@ and the row value."
                     (content (save-restriction
                                (org-narrow-to-subtree)
                                (org-drill-table--skip-props-and-schedule)
-                               (->> (buffer-substring (point) (point-max))
-                                 (s-split "\n")
-                                 (-drop 1)
-                                 (s-join "\n")
-                                 s-trim))))
+                               (->> (buffer-substring-no-properties (point) (point-max))
+                                    (s-split "\n")
+                                    (-drop 1)
+                                    (mapcar 's-trim)
```

Comment:
Use sharp quotes when referring to a function as a symbol, i.e. `(mapcar #'s-trim)`. This tells the byte-compiler that this function must be bound, and will emit a warning a compile-time if not.